### PR TITLE
Session 1 code

### PR DESCRIPTION
**Pull request by raduschirliu**

### CODE REVIEW

**Comment by raduschirliu** [comment]
May have made a bit of an oopsie here, as this branch also incorporated the changes to the main.py file - my bad

File: session1/main.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+# Useful colors 
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+
+# Creating the screen and the clock
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+clock = pygame.time.Clock()
+
+# Create our 50 x 100 rectangle surface at x = 0, y = 0
```
Comment (AlphaRLee) [inline session1/main.py:23]: As far as I understand, this technically isn't a surface
```suggestion
# Create our 50 x 100 rectangle at x = 0, y = 0
```

File: session1/main.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            pygame.quit()
+            sys.exit()
+        
+
+
+    """
+    UPDATE section - manipulate everything on the screen
+    """
+    
+    # Move our rectangle's x and y position by it's corresponding x and y speeds
```
Comment (AlphaRLee) [inline session1/main.py:43]: Grammar typo
```suggestion
    # Move our rectangle's x and y position by its corresponding x and y speeds
```

File: main_COPY_AND_RENAME.py [hunk 3]
```diff
@@ -7,20 +7,25 @@
...
+screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+clock = pygame.time.Clock()
 
 while True:
     """
     EVENTS section - how the code reacts when users do things
     """
     for event in pygame.event.get():
-        if event.type == pygame.QUIT:
+        if event.type == pygame.QUIT: # When user clicks the 'x' on the window, close our game
```
Comment (AlphaRLee) [inline main_COPY_AND_RENAME.py:28]: PEP8 says to double-space
```suggestion
        if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
```

File: session1/main.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+rect = pygame.Rect(0, 0, 50, 100)
+rect_speed_x = 7
+rect_speed_y = 5
+
+while True:
+    """
+    EVENTS section - how the code reacts when users do things
+    """
+    for event in pygame.event.get():
+        if event.type == pygame.QUIT: # When user clicks the 'x' on the window, close our game
```
Comment (AlphaRLee) [inline session1/main.py:33]: PEP8 says to double-space
```suggestion
        if event.type == pygame.QUIT:  # When user clicks the 'x' on the window, close our game
```

**Review by AlphaRLee — APPROVED** [review]
Looks good to me, just minor changes needed on comments. Feel free to merge after corrections are made

**Pull request merged**